spanner_orm/admin/scripts.py

```diff
@@ -42,11 +42,12 @@ def rollback(args: Any) -> None:
 def main(as_module: bool = False) -> None:
   prog = 'spanner-orm' if as_module else None
   parser = argparse.ArgumentParser(prog=prog)
+  # Subcommand is required, but please don't set required=true for
+  # compatibility with argparse < 3.7.
   subparsers = parser.add_subparsers(
       dest='subcommand',
       title='subcommands',
-      description='valid subcommands',
-      required=True)
+      description='valid subcommands')
 
   generate_parser = subparsers.add_parser(
       'generate', help='Generate a new migration')
```

Comment:
i can't leave this comment on the line i want to, but let's update the end of this method to be:
```
  args = parser.parse_args()
  if args.subcommand is None:
    parser.print_help()
  else:
    args.execute(args)
```
so that something useful is shown if no subcommands are specified (since the subcommand is no long required)